schemas/coverage.json

```diff
@@ -1,5 +1,13 @@
 {
     "$id" : "/schemas/coverage",
     "$ref": "/schemas/coverageBase",
-    "required" : [ "parameters" ]
+    "required" : [ "parameters" ],
+    "properties":
+    {
+        "domain":
+        {
+            "if": { "not": { "type": "string" } },
```

Comment:
Would `"if": { "type" : "object" }` be equivalent? It's a bit shorter and perhaps more explicit

Reply:
Yes, good point. I changed it.